google/transport.go

```diff
@@ -0,0 +1,176 @@
+package google
```

Comment:
Is this file auto-generated? If so, should it have a header?

Reply:
Nope, this one is hand-written.